src/render/tutorials/caching-http-requests/2_0.html.md

```diff
@@ -0,0 +1,314 @@
+      - "8092:8092"
+      - "18092:18092"
+    networks:
+      - knotnet
+```
+
+First and foremost it specifies Docker virtual network in which our services will be deployed. Then we have some service configs.
+The first one is `webapi`
+- `image` - docker image is provided here, 
+- `volumes` - directories to which Docker will have access, in our case it's directory where all Wiremock configs are stored,
+- `ports` - on the left side is provided port accessible from `localhost`, and on the right side we have port number available in virtual environment 
+- `networks` - specifies in which docker network, service will be deployed.
+
+Then we have `knotx` service specification, it's very similar to previous one, so only one property is worth to explain 
+- `command` - this property is used to point Knot.x starting commands.
+
+Next step in our configuration is to properly set up `settings.gradle.kts` (also at the project level directory) 
+```gradle
+rootProject.name = "api-cache"
+
+include("health-check")
+
+project(":health-check").projectDir = file("modules/health-check")
+``` 
+
+Here we have to specify project name, and packages to be included in this project. 
+
+Next thing to do is to configure `gradle.properties` at project level directory. Let's fill it with
+```gradle
+version=2.0.0-SNAPSHOT
+knotx.version=2.0.0
+knotx.conf=knotx
+docker.image.name=knotx-example/api-cache
+```
+Very important step here is to set `docker.image.name`, which is necessary to run proper docker image.
+
+`build.gradle.kts` file could stay as it was provided with starterkit, we don't have to apply any modifications here.
+The same if it comes to `./docker/Dockerfile`, version provided by default is absolutely enough for our purposes.
+
+## Wiremock 
+Wiremock is simply used for mocking server, we don't have to create real REST API, for our training purposes it's enough.
+To do it let's create following directories on the project level directory:
+- `common-services/webapi/__files/`
+- `common-services/webapi/mappings/`
+
+Into first directory please paste `product.json` file with following body:
+```json
+{
+  "id": 21762532,
+  "url": "http://knotx.io",
+  "label": "Product"
+}
+```
+
+And into second directory please paste `product.json` file with following body:
+```json
+{
+  "request": {
+    "method": "GET",
+    "url": "/product/id"
+  },
+  "response": {
+    "status": 200,
+    "fixedDelayMilliseconds": 100,
+    "bodyFileName": "product.json"
+  }
+}
+```
+
+Make sure that in `api-cache.yml` property `volume` properly points to `webapi` directory. And that's all about Wiremock.
+
+## Build and run
+From project level directory execute following commands which will build project and run Docker Swarm:
+```shell script
+$ gradlew clean build
+$ docker swarm init
+$ docker stack deploy -c api-cache.yml api-cache
+```
+After successful execution you should be able to see http response at [localhost:8092/product/id](localhost:8092/product/id)
```

Comment:
Some summary of what we did and what we can take from this would be nice. Like... moral of the story :D

Reply:
done